src/main/java/com/linecorp/armeria/server/docs/EnumInfo.java

```diff
@@ -22,14 +22,15 @@
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
+import java.util.Map;
 import java.util.Objects;
 
 import org.apache.thrift.meta_data.EnumMetaData;
 import org.apache.thrift.protocol.TType;
 
 class EnumInfo extends TypeInfo implements ClassInfo {
 
-    static EnumInfo of(EnumMetaData enumMetaData) {
+    static EnumInfo of(EnumMetaData enumMetaData, Map<String, String> docStrings) {
```

Comment:
Possible. So, how about `namespace` parameters? Is it okay for seperating previous factory method and docString supported factory method?

previous:
```
static StructInfo of(StructMetaData structMetaData) {
```

new factory method:
```
static StructInfo of(StructMetaData structMetaData, String namespace, Map<String, String> docStrings) {
```